micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/OutpatientMed.js

```
@@ -0,0 +1,38 @@
+export const OutpatientMed = ({ className, height = "30", width = "38", style = {}, fill = COLOR_FILL, onClick = null }) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      className={className}
+      onClick={onClick}
+      style={style}
+      viewBox="0 0 38 30"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)